### PR TITLE
Refresh Docker-compose and Helm Chart

### PR DESCRIPTION
**Pull request by BeArchiTek**

### CODE REVIEW

File: helm/Chart.yaml [hunk 1]
```diff
@@ -15,12 +15,12 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 4.0.2
+version: 4.1.0
```
Comment (BeArchiTek) [inline helm/Chart.yaml]: Not sure about this one @fatih-acar do you have a point of view ?

Comment (fatih-acar) [reply helm/Chart.yaml]: We only update the patch version when the infrahub version patch is bumped, so you can go with 4.0.3 I guess?

Comment (BeArchiTek) [inline helm/Chart.yaml:18]: Not sure about this one @fatih-acar do you have a point of view ?

Comment (BeArchiTek) [reply helm/Chart.yaml]: It was done with the invoke command, so gonna do it manually

Comment (fatih-acar) [reply helm/Chart.yaml]: Ok, I think we can leave it like that then, it's mostly cosmetic to me

**Review by fatih-acar — APPROVED** [review]

**Pull request merged**